preprocessors/semanticSeg/segment.py

```diff
@@ -105,9 +105,11 @@ def findContour(pred_color, width, height):
         centre_down = [centre_indi[0] / width, centre_indi[1] / height]
         area_down = area_indi / (width * height)
         contour_indi = contour_indi.tolist()
+        logging.info("Iterating through individual contour points with starting time: {}".format(str(time.time())))
         for j in range(len(contour_indi)):
             contour_indi[j][0] = float(float(contour_indi[j][0]) / width)
             contour_indi[j][1] = float(float(contour_indi[j][1]) / height)
+        logging.info("Iteration completed with time: {}".format(str(time.time())))
```

Comment:
Suggest `End contour iteration` so that it is clear what is ending. (see comment below about making start/end wording consistent)